### PR TITLE
Created key and cert file and updated server.js

### PR DESCRIPTION
**Pull request by dijodaiju7**
Created key and cert file and updated server.js
-> Generated key.pem and cert.pem
-> Created req.cnf
->Updated server.js

### CODE REVIEW

**Comment by sansyrox** [comment]
Hi @dijodaiju7 ,

Thank you for the work. But I can't see these in the commit and also there are a few conflicts. Please fix them. 
Thanks 😄 
> Generated key.pem and cert.pem

**Comment by dijodaiju7** [comment]
Can I know what's the issue?

**Comment by sansyrox** [comment]
@dijodaiju7 , I can't see the certificate files and there are some merge conflicts.

**Comment by sansyrox** [comment]
How are you generating the files?

**Comment by dijodaiju7** [comment]
I've generated key file, I'm sorry I forgot to generate the certificate, I'll fix that as soon as possible.

**Comment by sansyrox** [comment]
No worries. :D

File: server.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ const favicon = require('serve-favicon');
 const compression = require('compression');
 
 const app = express(),
-  // options = { 
-  //   key: fs.readFileSync(__dirname + '/key.pem'),
-  //   cert: fs.readFileSync(__dirname + '/cert.pem')
-  // },
+   options = {
```
Comment (sansyrox) [inline server.js]: I don't think you have used the options anywhere in the code. By themself, they are not of much use.

**Review by sansyrox — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (dijodaiju7) [reply server.js]: Ok

**Comment by dijodaiju7** [comment]
I've added certificate files

File: server.js (hunk 1 above)
Comment (sansyrox) [reply server.js]: Can you reset this to the original. I in works in a new config and your PR will help that significantly .

I will merge this afterwards :smile:

**Review by sansyrox — CHANGES_REQUESTED** [review]
Please remove `dist/main.js` as well. I will merge it after that.

**Comment by dijodaiju7** [comment]
I've done accordingly

**Comment by sansyrox** [comment]
Nicee. Good job 🔥

**Review by sansyrox — APPROVED** [review]

**Pull request merged**